runtime/compiler/runtime/J9CodeCacheManager.cpp

```diff
@@ -76,6 +77,36 @@ J9::CodeCacheManager::initialize(bool useConsolidatedCache, uint32_t numberOfCod
    return self()->OMR::CodeCacheManager::initialize(useConsolidatedCache, numberOfCodeCachesToCreateAtStartup);
    }
 
+bool
+J9::CodeCacheManager::isSufficientPhysicalMemoryAvailableForAllocation(size_t requestedCodeCacheSize)
+   {
+   TR::CodeCacheConfig &config = self()->codeCacheConfig();
+   bool incomplete;
+   // If swap memory is not available then no need to do further check
+   if (TR::Options::_isSwapMemoryAvailable)
+      {
+      if (config.verboseCodeCache())
+	 TR_VerboseLog::writeLineLocked(TR_Vlog_CODECACHE, "TR::Options::_isSwapMemoryAvailable = %s", TR::Options::_isSwapMemoryAvailable? "true":"false");
```

Comment:
We don't need to write this message every time a request for code cache allocations is made.

Reply:
True. This particular log was added for my tests purposes. Should not have been in the final fix. Will be removed.